demos/https/iot_demo_https_s3_download_async.c

```diff
@@ -888,8 +899,10 @@ static void _errorCallback( void * pPrivData,
                             IotHttpsReturnCode_t rc )
 {
     ( void ) reqHandle;
-    char * pRangeValueStr = ( ( _requestData_t * ) ( pPrivData ) )->pRangeValueStr;
-    IotLogError( "An error occurred during asynchronous operation with code: %d", pRangeValueStr, rc );
+    ( void ) respHandle;
+
+    char * pRangeValueStr = ( ( _requestContext_t * ) ( pPrivData ) )->pRangeValueStr;
```

Comment:
The API requires a non-NULL pPrivData?

Reply:
API does not require a non-null pPrivData, but I utilize the private data to give a useful log.